lib/daru/maths/statistics/vector.rb

```diff
@@ -700,7 +724,8 @@ def cumsum
         alias :std :sds
         alias :adp :average_deviation_population
         alias :cov :coefficient_of_variation
-        alias :variance :variance_sample    
+        alias :variance :variance_sample   
+        alias :covariance :covariance_sample  
```

Comment:
Alias for `variance` was already present, so won't it be a good idea to add for `covariance` as well?